onnx/defs/math/defs.cc

```diff
@@ -1029,6 +1029,48 @@ ONNX_OPERATOR_SET_SCHEMA(
             "Constrain input and output types to float tensors.")
         .TypeAndShapeInferenceFunction(propagateShapeAndTypeFromFirstInput));
 
+static const char* HardSwish_ver14_doc = R"DOC(
+HardSigmoid takes one input data (Tensor<T>) and produces one output data
+(Tensor<T>) where the HardSwish function, y = x * max(0, min(1, alpha * x + beta)),
```

Comment:
It would be helpful to relate this documentation with the function-body used below by adding an equivalent expression using HardSigmoid here. E.g., " = x * HardSigmoid<alpha=1/6, beta=0.5>(x)" or something like that.

Reply:
Done